Bottom: 7501f7ec80ad78f7636c3ef2dd1afcb911017b2f
Top:    54548cdcf3ba929962e7428236aff64d6e214cfd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 04:26:26 +0100

Refresh of create-new-function

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 760c6ef..2a8ae00 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -615,7 +615,7 @@ SwapSides (int argc, char **argv, Coord x, Coord y)
 {
   int old_shown_side = Settings.ShowSolderSide;
   int top_group = GetLayerGroupNumberBySide (TOP_SIDE);
-  int bottom_group = GetLayerGroupNumberBySide (BOTTOM_GROUP);
+  int bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
   int active_group = GetLayerGroupNumberByNumber (LayerStack[0]);
   int top_layer;
   int bottom_layer;
